### PR TITLE
Feature/unified UI enhancement/mob 1938 implement color pallet based default theme for alert

### PR DESCRIPTION
**Pull request by DavDo**
MOB 1938

**Jira issue:**
https://glia.atlassian.net/browse/MOB-1938

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Base.kt [hunk 1]
```diff
@@ -12,6 +12,10 @@ import com.glia.widgets.view.unifiedui.theme.base.*
 internal fun DefaultTheme(pallet: ColorPallet?): UnifiedTheme? = pallet?.let {
     UnifiedTheme(
         callTheme = CallDefaultTheme(it),
-        chatTheme = ChatDefaultTheme(it)
+        chatTheme = ChatDefaultTheme(it),
+        alertTheme = AlertDefaultTheme(it),
+        bubbleTheme = null,
+        surveyTheme = null,
+        callVisualizerTheme = null
```
Comment (gugalo) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Base.kt:19]: These are not going to be filled? I thought those would use the classes that you added. E.g: BubbleDefaultTheme

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import com.glia.widgets.view.unifiedui.theme.chat.UserImageTheme
+
+/**
+ * Default theme for Unread messages indicator
+ */
+internal fun BubbleDefaultTheme(pallet: ColorPallet): BubbleTheme? =
+    pallet.run {
+        val badge = BadgeDefaultTheme(this)
+        val userImage = UserImageDefaultTheme(this)
+        composeIfAtLeastOneNotNull(badge, userImage) {
```
Comment (gugalo) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt:19]: Did not know `composeIfAtLeastOneNotNull` method existed, could have used it in my last PR ))

File: widgetssdk/src/main/res/layout/chat_view.xml [hunk 3]
```diff
@@ -77,7 +77,7 @@
         android:layout_width="match_parent"
         android:layout_height="1dp"
         android:layout_marginTop="@dimen/glia_medium"
-        android:background="?attr/gliaBaseNormalColor"
+        android:background="?attr/gliaBaseShadeColor"
```
Comment (gugalo) [inline widgetssdk/src/main/res/layout/chat_view.xml:80]: Is this intended?

**Review by gugalo — APPROVED** [review]

File: widgetssdk/src/main/res/layout/chat_view.xml (hunk 3 above)
Comment (DavDo) [reply widgetssdk/src/main/res/layout/chat_view.xml:80]: Yes, I checked with ios and Figma, it was uncorrect before

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt (hunk 2 above)
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt:19]: I merged it a day before

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Base.kt (hunk 1 above)
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Base.kt:19]: will be fully filled on future PRs

**Review by gugalo — COMMENTED** [review]
There are also other commits included here from your other PRs, SHould we review only specific commits and not the whole PR?

**Review by andrews-moc — APPROVED** [review]

**Review by Pelkar — APPROVED** [review]
Looks solid

**Comment by DavDo** [comment]
!merge

**Pull request merged**